spec/02-integration/02-cmd/12-hybrid_spec.lua

```diff
@@ -129,6 +129,7 @@ for _, strategy in helpers.each_strategy() do
           cluster_control_plane = "127.0.0.1:9005",
           proxy_listen = "0.0.0.0:9002",
         }))
+        ngx.sleep(0.2)   -- wait for 'pids/nginx.pid'
```

Comment:
I think this could be done with a wait_until that checks that the pid file exists.